monitoring/definitions/worker.go

```diff
@@ -164,6 +164,23 @@ func Worker() *monitoring.Container {
 			shared.CodeInsights.NewInsightsQueryRunnerWorkerGroup(containerName),
 			shared.CodeInsights.NewInsightsQueryRunnerResetterGroup(containerName),
 			shared.CodeInsights.NewInsightsQueryRunnerStoreGroup(containerName),
+			{
+				Title:  "Code Insights Queue Utilization",
```

Comment:
```suggestion
				Title:  "Codeinsights: queue utilization",
```

nit - a bit of standardization with other naming

![image](https://user-images.githubusercontent.com/23356519/134252597-f233fc69-33f6-4513-9226-dff2d2524f4c.png)